examples/browser-to-server/tests/test.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+import { test, expect } from '@playwright/test'
+import { playwright } from 'test-util-ipfs-example'
+import { spawn, exec } from 'child_process'
+import { existsSync } from 'fs'
+
+// Setup
+const play = test.extend({
+  ...playwright.servers()
+})
+
+async function spinUpGoLibp2p() {
```

Comment:
nit: spawnGoLibp2p?

Reply:
Good name change, this has been updated.